src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -92,22 +94,65 @@ class WorkflowManagerActor(backend: Backend) extends Actor with CromwellActor {
     restartIncompleteWorkflows()
   }
 
+  private def reply[A](id: WorkflowId, fa: Future[A],
+                       onSuccess: (WorkflowId, A) => WorkflowManagerSuccessResponse,
+                       onFailure: (WorkflowId, Throwable) => WorkflowManagerFailureResponse): Unit = {
+    // Don't close over sender.
+    val sndr = sender
+    fa onComplete {
+      case Success(a) => sndr ! onSuccess(id, a)
+      case Failure(e) => sndr ! onFailure(id, e)
+    }
+  }
+
+  private def reply[A, B](id: WorkflowId, a: A, fb: Future[B],
+                          onSuccess: (WorkflowId, A, B) => WorkflowManagerSuccessResponse,
+                          onFailure: (WorkflowId, A, Throwable) => WorkflowManagerFailureResponse): Unit = {
+    // Don't close over sender.
+    val sndr = sender
```

Comment:
Very minor preference to `sender()`, to let the reader know that this isn't a value, but a method. Because it's a method, that's part of the reason the call to sender shouldn't be wrapped in a closure.